pages/sections/landing.tsx

```diff
@@ -10,7 +10,215 @@ import SignupCounter from '@/components/SignupCounter';
 import Link from 'next/link';
 import { useRouter } from 'next/router';
 import Image from 'next/image';
-import Skyline from '../../public/assets/landing.svg';
```

Comment:
sorry for scope creepin, but marketing wants the landing 'skyline' to be shifted a lil up? like when u first load the website, the skyline is cutoff.

it looks like dis
<img width="1510" alt="image" src="https://user-images.githubusercontent.com/62922309/228082357-fcd0f2a7-b55f-42f8-83d6-a608486900e5.png">

but it shud look like dis
<img width="1512" alt="image" src="https://user-images.githubusercontent.com/62922309/228082393-b45e76c4-e159-43c8-ab1a-4ea42f0a7043.png">